context_test.go

```diff
@@ -33,24 +33,28 @@ import (
 	"github.com/uber/jaeger-client-go"
 )
 
-func TestContext_HostAccess(t *testing.T) {
-	ctx := New(gcontext.Background(), service.NopHost())
+type testKey int
+
+const (
+	_testContextKey testKey = iota
+	_testContextKey2
+)
+
+func TestContext_LoggerAccess(t *testing.T) {
+	ctx := SetContextStore(context.Background(), service.NopHost())
 	assert.NotNil(t, ctx)
-	assert.NotNil(t, ctx.Logger())
-	assert.NotNil(t, ctx.Value(_fxContextStore))
+	assert.NotNil(t, Logger(ctx))
+	assert.NotNil(t, ctx.Value(_contextStore))
 }
 
-func TestWithContext(t *testing.T) {
-	gctx := gcontext.WithValue(gcontext.Background(), "key", "val")
-	ctx := New(gctx, service.NopHost())
-	assert.Equal(t, "val", ctx.Value("key"))
-
-	gctx1 := gcontext.WithValue(gcontext.Background(), "key1", "val1")
-	ctx = &Context{
-		Context: gctx1,
-	}
-	assert.Equal(t, nil, ctx.Value("key"))
-	assert.Equal(t, "val1", ctx.Value("key1"))
+func TestContext_WithLogger(t *testing.T) {
```

Comment:
hmm. I just ported the internal tests out. This was testing if we get the keys out of context. I'll remove it.